README.md

```diff
@@ -1,17 +1,38 @@
-# rpm-ostree Overview
-
-rpm-ostree is a hybrid image/package system.  It uses
-[OSTree](https://ostree.readthedocs.io/en/latest/) as a base image
-format, and supports RPM on both the client and server side using
-[libdnf](https://github.com/rpm-software-management/libdnf).
+# rpm-ostree: A true hybrid image/package system
+
+```
+                         +-----------------------------------------+
+                         |                                         |
+                         |       rpm-ostree (daemon + CLI)         |
+                  +------>                                         <---------+
+                  |      |     status, upgrade, rollback,          |         |
+                  |      |     pkg layering, initramfs --enable    |         |
+                  |      |                                         |         |
+                  |      +-----------------------------------------+         |
+                  |                                                          |
+                  |                                                          |
+                  |                                                          |
++-----------------|-------------------------+        +-----------------------|-----------------+
+|                                           |        |                                         |
+|         libostree (image system)          |        |            libdnf (pkg system)          |
+|                                           |        |                                         |
+|   C API, hardlink fs trees, system repo,  |        |    ties together libsolv (SAT solver)   |
+|   commits, atomic bootloader swap         |        |    with librepo (RPM repo downloads)    |
+|                                           |        |                                         |
++-------------------------------------------+        +-----------------------------------------+
```

Comment:
Ouuh, nice ASCII art! I hope you used a tool to build this. :)

Also, let's put it after the introductory paragraph instead?

Reply:
Yeah, http://asciiflow.com/ though I have mixed feelings about it.